drivers/gpio/gpio_nrf5.c

```diff
@@ -375,6 +383,28 @@ static const struct gpio_driver_api gpio_nrf5_drv_api_funcs = {
 	.disable_callback = gpio_nrf5_disable_callback,
 };
 
+/* Enable GPIOTE Interrupt */
+void nrf_gpiote_interrupt_enable(uint32_t mask)
```

Comment:
This function is not needed (and use of extern in caller c files). Use nrf_gpiote_int_enable instead and just a function (if not already exists) to clear EVENT, called before nrf_gpiote_int_enable is called by caller code.

Reply:
nrf_gpiote_interrupt_enable() is called by application code whereas nrf_gpiote_int_enable is present as a HAL function (static inline inside ext/hal/nordic/hal/nrf_gpiote.h).  It is not recommended to call directly as HAL function from sample application. 
Also this puts sort of abstraction layer for applications developer so that she need not worry in case HAL is changing later point.

Reply:
ok, agree. No change required here.